<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>ChangeBorgPassphrase</class>
  <widget class="QDialog" name="ChangeRepositoryPass">
    <property name="modal">
      <bool>true</bool>
    </property>
    <layout class="QGridLayout" name="gridLayout">
      <property name="verticalSpacing">
        <number>0</number>
      </property>

      <item row="2" column="0">
        <layout class="QFormLayout" name="repoDataFormLayout">
          <property name="fieldGrowthPolicy">
            <enum>QFormLayout::ExpandingFieldsGrow</enum>
          </property>
          <property name="leftMargin">
            <number>5</number>
          </property>
          <property name="topMargin">
            <number>5</number>
          </property>
          <property name="rightMargin">
            <number>5</number>
          </property>
          <property name="bottomMargin">
            <number>20</number>
          </property>

          <item row="4" column="0" colspan="2">
            <widget class="QLabel" name="errorText">
              <property name="text">
                <string></string>
              </property>
            </widget>
          </item>

        </layout>
      </item>
      <item row="3" column="0">
        <widget class="QDialogButtonBox" name="buttonBox">
          <property name="standardButtons">
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <resources />
  <connections />
</ui>
